### PR TITLE
Calendar Service integration

### PR DESCRIPTION
**Pull request by yinan-liu**
1. proactive-server startup script was modified to exlude calendar-service*.jar from CLASSPATH
2. calendar-service-setup scripts for windows and linux. They are used to install Radicale server and python3 lib
3. calendar-service scripts for windows and linux. They are used to create user accounts in Radicale server based on the scheduler login.cfg file, and launch the calendar service application
4. radicale folder contains the pre-defined radicale server config and the radicale server library for windows and linux

### CODE REVIEW

**Comment by marcocast** [comment]
Seems ok to me

File: tools/calendar-service.bat [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+:fail
+@rem Set variable ${exitEnvironmentVar} if you need the _script_ return code instead of
+@rem the _cmd.exe /c_ return code!
+if  not "" == "%${exitEnvironmentVar}%" exit 1
+exit /b 1
+
+:mainEnd
+if "%OS%"=="Windows_NT" endlocal
+
+:omega
```
Comment (mbenguig) [inline tools/calendar-service.bat]: ?

**Comment by mbenguig** [comment]
Just 1 comment.. and it is OK!

File: tools/calendar-service.bat (hunk 1 above)
Comment (yinan-liu) [inline tools/calendar-service.bat]: ok, remove it

**Pull request merged**